.github/workflows/pr-tests.yml

```diff
@@ -112,7 +112,7 @@ jobs:
 
       - name: Run Playwright tests locally for Preview
         if: ${{ github.event_name == 'pull_request' }}
-        run: xvfb-run --auto-servernum -- pnpm --filter e2e-tests test:e2e
+        run: xvfb-run --auto-servernum -- pnpm --filter @e2e-tests/runner test:e2e
         env:
```

Comment:
We could have all these envs in the runner's .env.example, so it's easier for anyone to run them theirselves when needed.